src/Elcodi/Admin/BannerBundle/AdminBannerBundle.php

```diff
@@ -17,6 +17,7 @@
 
 namespace Elcodi\Admin\BannerBundle;
 
+use Symfony\Component\Console\Application;
```

Comment:
I (realy) prefer a trait as you can extend from one (and only one) class, this is more about sharing a function than create a filiation between a mother class and children imo :)